README.md

```diff
@@ -107,17 +108,24 @@ accounts available between them.*
 
 ### Full node on the Rinkeby test network
 
-The above test network is a cross-client one based on the ethash proof-of-work consensus
-algorithm. As such, it has certain extra overhead and is more susceptible to reorganization
-attacks due to the network's low difficulty/security. Go Ethereum also supports connecting
-to a proof-of-authority based test network called [*Rinkeby*](https://www.rinkeby.io)
-(operated by members of the community). This network is lighter, more secure, but is only
-supported by go-ethereum.
+Go Ethereum also supports connecting to the older proof-of-authority based test network 
+called [*Rinkeby*](https://www.rinkeby.io) which is operated by members of the community.
 
 ```shell
 $ geth --rinkeby console
 ```
 
+### Full node on the Ropsten test network
+
+In addition to Görli and Rinkeby, Go-Ethereum also supports the ancient Ropsten testnet.
+The Ropsten test network is based on the Ethash proof-of-work consensus algorithm. As such,
+it has certain extra overhead and is more susceptible to reorganization attacks due to the
+network's low difficulty/security. 
+
+```shell
+$ geth --ropsten console
+```
```

Comment:
Perhaps also add a note that for legacy reasons, this uses the `testnet` folder on disk, not the `ropsten`. Alternatively, we need a migration path (if the old `testnet` path exists, use that, otherwise use `ropsten`).